### PR TITLE
chime: Support playing motion test sound

### PR DESCRIPTION
**Pull request by vickyg3**
Changes the test_sound property to be a function.

Fixes Issue #28.

Please let me know if this change needs to be backward compatible. I can make it another property in that case. (Also thanks for the pointer on mitmproxy, worked like a charm!)

### CODE REVIEW

File: ring_doorbell/const.py [hunk 1]
```diff
@@ -40,6 +40,11 @@
 URL_DOORBELL_HISTORY = DOORBELLS_ENDPOINT + '/history'
 URL_RECORDING = '/clients_api/dings/{0}/recording'
 
+# chime test sound kinds
+KIND_DING = 'ding'
+KIND_MOTION = 'motion'
+CHIME_TEST_SOUND_KINDS = (KIND_DING, KIND_MOTION)
```
Comment (tchellomello) [inline ring_doorbell/const.py]: Let's make a list instead a map

File: ring_doorbell/__init__.py [hunk 2]
```diff
@@ -388,11 +388,12 @@ def linked_tree(self):
...
 
-    @property
-    def test_sound(self):
+    def test_sound(self, kind=KIND_DING):
         """Play chime to test sound."""
+        if kind not in CHIME_TEST_SOUND_KINDS:
+            return False
         url = API_URI + TESTSOUND_CHIME_ENDPOINT.format(self.account_id)
-        self._ring.query(url, method='POST')
+        self._ring.query(url, method='POST', extra_params={"kind": kind})
```
Comment (tchellomello) [inline ring_doorbell/__init__.py]: Shouldn't this be expanded to the "ringtone" the user selected to play?

Comment (vickyg3) [reply ring_doorbell/__init__.py]: It probably should be. But i'll leave that to a follow-up. Don't have my phone set-up to do the proxy again right now. :)

File: ring_doorbell/const.py (hunk 1 above)
Comment (vickyg3) [reply ring_doorbell/const.py]: I'm sorry, but can you please explain what the key and value of the map would be?

Comment (tchellomello) [reply ring_doorbell/const.py]:
```python
CHIME_TEST_SOUND_KINDS = [KIND_DING, KIND_MOTION]
```

Comment (vickyg3) [reply ring_doorbell/const.py]: Why does it have to be a list? What i've used is a tuple which is basically an immutable list (not a map). Which is exactly what we need here.

File: ring_doorbell/__init__.py (hunk 2 above)
Comment (tchellomello) [reply ring_doorbell/__init__.py]: Yes make sense.. You got a good point!!

**Pull request merged**